c/termination-crafted/LexIndexValue-Pointer-3.c

```diff
@@ -15,6 +15,9 @@ extern int __VERIFIER_nondet_int(void);
 
 int main() {
 	int *p = malloc(1048 * sizeof(int));
+	for (int i = 0; i < 1048; ++i) {
+		p[i] = __VERIFIER_nondet_int();
+	}
```

Comment:
Why is the initialization with nondet value necessary? As far as I can see it is a valid c program without initializing pointer values.

Reply:
Reading heap-allocated memory without an intervening write is undefined behaviour (see https://stackoverflow.com/a/9220049 for an elaboration).